blocks/job-list/job-list.css

```diff
@@ -0,0 +1,63 @@
+.job-list {
+  --color-text: var(--spectrum-gray-900);
+  --color-border: var(--spectrum-gray-400);
+  margin-block: 2rem;
+}
+
+.job-list__department {
+  font-size: var(--spectrum-heading-size-s);
+  margin-block: 0 2rem;
+  color: var(--color-text);
+
+  @media (min-width: 48rem) {
+    font-size: var(--spectrum-heading-size-m);
+  }
+
+  @media (min-width: 80rem) {
+    font-size: var(--spectrum-heading-size-s);
+  }
+}
+
+.job-list__listings {
+  display: grid;
+  gap: 0;
+
+  & .job-listing:first-child {
+    border-top: none;
+  }
+
+  & .job-listing:last-child {
+    border-bottom: none;
+  }
+}
+
+@media (min-width: 48rem) {
+  .job-list__listings .job-listing::after {
+    display: none;
+  }
+}
+
+@media (min-width: 80rem) {
+  .job-list__listings {
+    display: grid;
+    grid-template-columns: repeat(2, minmax(0, 32rem));
```

Comment:
Did we choose custom grids for a reason? If not, there are some grid classes in the `styles` directory, you can see an example on the section-header pr here: https://github.com/adobe/adobe-design-website/pull/71/files#diff-dd533ac34457a09ecf467467842896fcb2fb61800ef9bfdd906716cee2b9aa10R30